foundationdb-kubernetes-sidecar/sidecar.py

```diff
@@ -483,21 +489,12 @@ def do_POST(self):
         This method executes a POST request.
         """
         try:
-            if not self.check_request_cert():
+            if not self.check_request_cert(self.path):
                 return
             if self.path == "/copy_files":
                 self.send_text(copy_files())
-            elif self.path == "/copy_binaries":
-                self.send_text(copy_binaries())
-            elif self.path == "/copy_libraries":
-                self.send_text(copy_libraries())
```

Comment:
I'm not clear on why we're getting rid of these endpoints.

Reply:
That's something separate. The operator doesn't use these endpoints but we can also leave them as they are.

Reply:
I'd rather have a separate discussion. Even though the operator doesn't use these endpoints, there's a possibility that someone else would, and we might want them to be available in some debugging scenarios.